docs/gitrepo-add.md

```diff
@@ -168,6 +168,40 @@ For example, to add a secret in kubectl, run
 
 After secret is created, specify the secret to `gitRepo.spec.helmSecretName`. Make sure secret is created under the same namespace with gitrepo.
 
+### Use different helm credentials for each path
+
+:::info
+`gitRepo.spec.helmSecretName` will be ignored if `gitRepo.spec.helmSecretNameForPaths` is provided
+:::
+
+Create a file `secrets-path.yaml` that contains credentials for each path defined in a `GitRepo`. Credentials will not be used
+for paths that are not present in this file.
+The path is the actual path to the bundle within the git repository, which might have more segments than the entry under `paths:`. 
```

Comment:
This line reads a bit confusing to me; are readers of this document expected to know what a _bundle_ is in this context, considering that it's a [lower-level concept](https://fleet.rancher.io/bundle-add) than a `GitRepo`? By `the actual path to the bundle`, do we mean a path to a `fleet.yaml`?
Would an example help perhaps?
ie, do we mean that if a `GitRepo` spec contains:
```
spec:
  repo: http://foo.com/bar
  paths:
  - baz/my_path
```
... then `secrets-path.yaml` could look like:
```
baz/my_path/dir_where_fleet_yaml_lives: # more segments than in the `paths` entry above
  username: user
  password: pass
```
?

Happy to discuss, as I'm not yet sure that I understand this.

Reply:
In this case `secrets-path.yaml` could look like
```
baz/my_path: # more segments than in the `paths` entry above
  username: user
  password: pass
```
Fleet will create a Bundle for each dir that has a fleet.yaml. More info https://fleet.rancher.io/gitrepo-content#how-repos-are-scanned

If any subdirectory of `baz/my_path` contains a `fleet.yaml` a new Bundle will be created.   `secrets-path.yaml`  for this subdir would look like:
```
baz/my_path/subdir: # more segments than in the `paths` entry above
  username: user
  password: pass
```

Not sure how to make this clearer, maybe a link to https://fleet.rancher.io/gitrepo-content#how-repos-are-scanned before so users can see how Bundles are created?
Any other suggestion is very welcome

Reply:
Thanks, this is clearer to me; I think we could:
* either add a link to https://fleet.rancher.io/gitrepo-content#how-repos-are-scanned, as you're suggesting
* or simply add a few words to say that the path to a bundle is the path to a `fleet.yaml` file, eg.:
```suggestion
The path is the actual path to the bundle (ie to a folder containing a `fleet.yaml` file) within the git repository, which might have more segments than the entry under `paths:`. 
```
or even:
```suggestion
The path is the actual path to the bundle (ie to a folder containing a `fleet.yaml` file) within the git repository, which might be a subfolder of an entry under `paths:`. 
```